razorpay-payments.php

```diff
@@ -102,7 +108,14 @@ function init_form_fields()
                         'authorize' => 'Authorize',
                         'capture'   => 'Authorize and Capture'
                     )
-                )
+                ),
+                'enable_webhook' => array(
+                    'title' => __('Enable Webhook', 'razorpay'),
+                    'type' => 'checkbox',
+                    'description' => esc_url( admin_url('admin-post.php') ) . '?action=rzp_webhook',
+                    'label' => __('Enable Razorpay Webhook with the URL listed below.', 'razorpay'),
+                    'default' => 'yes'
+                ),
```

Comment:
We need to add instructions about how to configure webhooks on the website. Something like:

>Use https://www.woocommerce.com/adlkasd/asfdsaf.php as the Webhook URL [here](https://dashboard.razorpay.com/#/app/webhooks) and enable the `payment.authorized` event _only_.

We will also need Signature verification in place (add another input for webhook secret (minlength=8))